scheduler/feasible.go

```diff
@@ -223,13 +233,26 @@ func (h *HostVolumeChecker) hasVolumes(n *structs.Node) bool {
 			}
 
 			if req.Sticky {
-				if slices.Contains(h.hostVolumeIDs, vol.ID) || len(h.hostVolumeIDs) == 0 {
+				// the node is feasible if there are no claims or if there's an
+				// exact match
```

Comment:
I'd probably word this as "no remaining claims to fulfill", as it makes the intent of the deletion more clear for future readers.